client/lib/plans/features-list.js

```diff
@@ -650,7 +650,7 @@ export const FEATURES_LIST = {
 
 	[ constants.FEATURE_STANDARD_SECURITY_TOOLS ]: {
 		getSlug: () => constants.FEATURE_STANDARD_SECURITY_TOOLS,
-		getTitle: () => i18n.translate( 'Standard Security Tools' ),
+		getTitle: () => i18n.translate( 'Standard security tools' ),
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.